app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java

```diff
@@ -739,10 +752,19 @@ public void onPos(final SourceContent sourceContent, boolean b) {
                         public void onClick(View view) {
                             linkPreviewViewHolder.onClick(view);
                             Uri webpage = Uri.parse(sourceContent.getFinalUrl());
-                            Intent intent = new Intent(Intent.ACTION_VIEW, webpage);
-                            if (intent.resolveActivity(currContext.getPackageManager()) != null) {
-                                currContext.startActivity(intent);
-                            }
+
+
+                            CustomTabsIntent.Builder intentBuilder = new CustomTabsIntent.Builder();
+
+                            intentBuilder.setToolbarColor(ContextCompat.getColor(context, R.color.colorPrimary));
+                            intentBuilder.setSecondaryToolbarColor(ContextCompat.getColor(context, R.color.colorAccent));
+
+                            intentBuilder.setExitAnimations(context, android.R.anim.slide_in_left,
+                                    android.R.anim.slide_out_right);
```

Comment:
This code is used multiple times, it would be best if you could define it under a function and call the function each time you need it. Can you do that please? --- Reuabability is awesome.